src/helpers.ts

```diff
@@ -8,8 +8,10 @@ import { SubscriptionClient } from './client';
 // Quick way to add the subscribe and unsubscribe functions to the network interface
 // We will move this into a new package in the future
 export function addGraphQLSubscriptions(networkInterface: any, wsClient: SubscriptionClient): any {
-  console.warn('This method becomes deprecated in the new package graphql-transport-ws. Start using the ' +
-    'GraphQLTransportWSClient to make queries, mutations and subscriptions over websockets.');
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
yeah, ask him what we did last time, i just don't remember but i think now apollo has some approach about it, if he is not reachable you can search production in either apollo-client or apollo-angular it happend on one of them